code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
It takes 6 seconds to stun with a stunprod/baton and cuff. It takes 6 seconds to kill a man with a normal sec laser (get yer spam on).

Imagine this. You want to use this to knock a officer out and steal his id. We are using pure Somnistimine and a syringe gun aka 15 units. Breaking down the fight in stages. 

First second: Run behind a corner and fire. Average human reaction time is 0.25 seconds, but lets give it a second.

Second two: Officer notices the syringe and gets hit. You dodge behind the corner.

Third second: Officer takes cover and writes in caps MEDBAY MAINT. (Should take half a second to write that).

Forth second: Officer draws taser. You still behind a corner.

Fifth second: Officer rushes you while you try to run. 

Sixth second: He catches up and fires a taser.

Seventh second: You get hit. Go down and are stunned for 6 cycles aka 12 seconds. Officer finally falls. As you lie on the ground sobbing and hoping HonkMother gives you RNG luck, other officers are rushing towards you. 

(Extra section time! Since you fired 15 units of Somi at officer, it takes 15:0,4 (meta rate)=37,4 cycles (64 seconds roughly) to fully get rid of it. 64-6= 58 aka hes stunned for 58 seconds. During that time, he can give others your name, what you wear and more detailed location.